application/Config/Cache.php

```diff
@@ -101,6 +101,21 @@ class Cache extends BaseConfig
 	];
 
 	/*
+	| -------------------------------------------------------------------------
+	| Predis settings
+	| -------------------------------------------------------------------------
+	| Your Redis server can be specified below, if you are using
+	| the Predis drivers.
+	|
+	*/
+    public $predis = [
```

Comment:
I would say let's scrap the `predis` specific config and just use the `redis` config, unless you know of a specific reason not to?